scss/components/_card.scss

```diff
@@ -99,6 +104,10 @@ $card-margin: $global-margin !default;
     flex: 1 0 auto;
   }
 
+  border-width: 1px;
+  border-style: none inset solid outset;
+  border-color: $light-gray;
```

Comment:
Same as before, if you want to visually put border on Card, do not apply them on one of its inner elements.

Here what it gives for a single Card:
To make them more visible, `.card` borders are in green, `.card-section` borders are in red.

![screen shot 2017-01-19 at 13 36 53](https://cloud.githubusercontent.com/assets/9939075/22107096/6137e302-de4c-11e6-9c99-add2bfec9ab9.png)